engineering_blogs.opml

```diff
@@ -115,7 +115,6 @@
       <outline type="rss" text="Flickr" title="Flickr" xmlUrl="http://code.flickr.net/feed/" htmlUrl="http://code.flickr.net/"/>
       <outline type="rss" text="Flipboard" title="Flipboard" xmlUrl="http://engineering.flipboard.com/feed.xml" htmlUrl="http://engineering.flipboard.com/"/>
       <outline type="rss" text="Foursquare" title="Foursquare" xmlUrl="https://engineering.foursquare.com/feed" htmlUrl="https://engineering.foursquare.com/"/>
-      <outline type="rss" text="Freeletics" title="Freeletics" xmlUrl="https://freeletics.engineering/feed.xml" htmlUrl="https://freeletics.engineering/"/>
```

Comment:
This should not have been removed

Reply:
Freeletics's Engineering blog isn't on the README, thats probably why it was removed. A separate PR to include Freeletics again should resolve this, what do you think?

Reply:
Sounds good, would you like to submit it? 😄

Reply:
Sure! 👍  I'll make a separate PR Freeletics soon.